### PR TITLE
godep update

### PR DESCRIPTION
**Pull request by gmarek**
cc @brendandburns @thockin @davidopp @lavalamp @dchen1107 @smarterclayton @xiang90

### CODE REVIEW

**Comment by smarterclayton** [comment]
Is this to get the latest heapster?  Or something else?

**Comment by gmarek** [comment]
I was trying to add a new thing that depended on kubernetes main to contrib. This is just a byproduct of trying to make dependencies work.

I failed in the end, as there's just too much incompatibility out there, so I'll just create a new project.

**Comment by smarterclayton** [comment]
ok to test

**Comment by piosz** [comment]
Un-assigning myself from this PRs since I'm not an expert in godeps. @smarterclayton do you want to review?

**Comment by piosz** [comment]
Un-assigning myself from this PRs since I'm not an expert in godeps. @smarterclayton do you want to review?

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -896,6 +909,10 @@
 			"Rev": "c2528b2dd8352441850638a8bb678c2ad056fd3e"
 		},
 		{
+			"ImportPath": "golang.org/x/net/http2",
```
Comment (smarterclayton) [inline Godeps/Godeps.json]: What uses this?  In go 1.6 this is part of stdlib, was this a dependency we weren't taking before from grpc?  Is the bradfitz/http version no longer used?

Comment (gmarek) [inline Godeps/Godeps.json]: I have no idea - this is what I got after `godep save ./...`

Comment (smarterclayton) [inline Godeps/Godeps.json]: It *looks* like the only thing that depended on bradfitz/http2 was the old version of grpc.  If the new version of GRPC is depending on net/http2, I think we no longer need the old version.  Can you check the dependency tree locally?

Comment (gmarek) [inline Godeps/Godeps.json]: Yeah - I think that the new grpc version wasn't compatible with something in contrib, and I tried to write some code that will be able to import stuff from pkg/kubelet. I probably had to grab old grpc version for that. 

I'm probably going to create a separate repo in main k8s.io one and drop this PR, as I don't think it's worth trying to maintain dependencies of contrib...

**Pull request closed without merging**